### PR TITLE
Deploy certified operator with kind / github CI

### PR DESCRIPTION
**Pull request by edcdavid**
Deploys a community operator that is also redhat certified: there is only one: zoperator ! However it is certified only for openshift 4.7 so the tnfsuite operator certification will fail for openshift versions != 4.7
GitHub CI test will pass since there is no openshift version to compare to.
Workflow:
- install OLM
- create an operator group in tnf namespace 
- create a subscription
- install plan and csv are created

### CODE REVIEW

**Review by sebrandon1 — APPROVED** [review]
Just a couple of comments but nothing that breaks the current PR.  Approved!

File: scripts/init-env.sh [hunk 1]
```diff
@@ -75,4 +78,5 @@ if [ $NUM -ge 0 ]; then
   export MULTUS_ANNOTATION="'[ ${MULTUS_ANNOTATION::-1} ]'"
 fi
 
-export 
\ No newline at end of file
+COMMUNITY_OPERATOR_NAME=zoperator.v0.3.6
```
[the remark this answers is not in the document]
Comment (edcdavid) [reply scripts/init-env.sh]: fixed

File: scripts/clean.sh [hunk 2]
```diff
@@ -12,3 +12,4 @@ source $SCRIPT_DIR/init-env.sh
 ./$SCRIPT_DIR/delete-test-crds.sh
 ./$SCRIPT_DIR/delete-debug-ds.sh
 ./$SCRIPT_DIR/delete-statefulset-pods.sh
+./$SCRIPT_DIR/delete-community-operator.sh
```
Comment (greyerof) [inline scripts/clean.sh:15]: Shouldn't you also remove the delete-operator.sh (line 8)?

File: scripts/deploy-community-operator.sh [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+rm ./temp/rendered-local-community-operator-group.yaml
+
+# Create the Subscription
+mkdir -p ./temp
+cat ./test-target/community-operator-subscription.yaml | OPERATOR_NAME=$COMMUNITY_OPERATOR_BASE CATALOG_SOURCE=$CATALOG_SOURCE CATALOG_NAMESPACE=$CATALOG_NAMESPACE TNF_EXAMPLE_CNF_NAMESPACE=$TNF_EXAMPLE_CNF_NAMESPACE $SCRIPT_DIR/mo > ./temp/rendered-local-community-operator-subscription.yaml
+oc apply -f ./temp/rendered-local-community-operator-subscription.yaml
+rm ./temp/rendered-local-community-operator-subscription.yaml
+
+
+TIMEOUT=24 # 240 seconds
```
[the remark this answers is not in the document]
Comment (edcdavid) [reply scripts/deploy-community-operator.sh]: fixed

File: scripts/clean.sh (hunk 2 above)
Comment (edcdavid) [reply scripts/clean.sh:14]: fixed

File: scripts/deploy-community-operator.sh [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+mkdir -p ./temp
+cat ./test-target/community-operator-subscription.yaml | OPERATOR_NAME=$COMMUNITY_OPERATOR_BASE CATALOG_SOURCE=$CATALOG_SOURCE CATALOG_NAMESPACE=$CATALOG_NAMESPACE TNF_EXAMPLE_CNF_NAMESPACE=$TNF_EXAMPLE_CNF_NAMESPACE $SCRIPT_DIR/mo > ./temp/rendered-local-community-operator-subscription.yaml
+oc apply -f ./temp/rendered-local-community-operator-subscription.yaml
+rm ./temp/rendered-local-community-operator-subscription.yaml
+
+
+CSV_CHECK_RETRIES=24 # 240 seconds
+while [[ $(oc get csv -n $TNF_EXAMPLE_CNF_NAMESPACE $COMMUNITY_OPERATOR_NAME -o go-template="{{.status.phase}}") != "Succeeded" && "$CSV_CHECK_RETRIES" -gt 0 ]]; do
+        echo "waiting for $COMMUNITY_OPERATOR_NAME installation to succeed"
+        sleep 10
```
Comment (greyerof) [inline scripts/deploy-community-operator.sh:41]: Please fix indentation of lines 40 and 41.

**Review by greyerof — APPROVED** [review]

File: scripts/deploy-community-operator.sh (hunk 4 above)
Comment (edcdavid) [reply scripts/deploy-community-operator.sh]: fixed

**Pull request merged**